file_handlers.py

```diff
@@ -421,9 +421,9 @@ def write_entry_handler(syscall_id, syscall_object, pid):
     bytes_from_trace = bytes_from_trace.decode('string-escape')
     logging.debug(bytes_from_trace.encode('hex'))
     logging.debug(bytes_from_execution.encode('hex'))
-    # if bytes_from_trace != bytes_from_execution:
```

Comment:
I'd take this change out for now as it doesn't relate to the rt_sigaction stuff.  I'll push this fix separately